sys/arm64/arm64/locore.S

```diff
@@ -686,7 +686,8 @@ build_l1_block_pagetable:
 	/* Build the L1 block entry */
 	orr	x12, x7, #L1_BLOCK
 #if __has_feature(capabilities)
-	orr	x12, x12, #(ATTR_CAP_RW)
+	orr	x12, x12, #(ATTR_LC_ENABLED)
```

Comment:
Does this require two instructions or can it be `#(ATTR_LC_ENABLED | ATTR_CDBM)`?